flask_sse.py

```diff
@@ -4,6 +4,7 @@
 from collections import OrderedDict
 from flask import Blueprint, request, current_app, json, stream_with_context
 from redis import StrictRedis
+from redis import exceptions as redis_exceptions
```

Comment:
You can probably just do:
```suggestion
from redis.exceptions import ConnectionError
```